changelog.d/4528.feature

```diff
@@ -0,0 +1 @@
+Synapse can now automatically provision TLS certificates via ACME (the protocol used by CAs like Let's Encrypt).
```

Comment:
I fear this needs to say something different given it's going in post-RC

Reply:
does it? i'm confused

Reply:
oh

Reply:
done